src/palette/qml/MuseScore/Palette/internal/PaletteTree.qml

```diff
@@ -47,6 +47,8 @@ StyledListView {
     property string filter: ""
     property bool searchOpened: false
 
+    property bool isResultFound: true
```

Comment:
I would recommend declaring this property as follows:
isResultFound: count === 0

You can remove onCountChanged after this change

Reply:
Thanks for the feedback! I changed to `isResultFound: count !== 0` and removed `onCountChanged` to keep `PalettesPanel.qml` untouched.